static/app/views/dashboards/widgets/lineChartWidget/lineChartWidget.stories.tsx

```diff
@@ -88,3 +110,28 @@ export default storyBook(LineChartWidget, story => {
 const MediumWidget = styled('div')`
   width: 420px;
 `;
+
+function filterTimeSeriesToSelection(
```

Comment:
A big thank you for making this a separate function and not defining it inside the component body, this is a big part in how we make components more readable ❤️